src/BuiltinExtensions/ComfyUIBackend/WorkflowGeneratorModelSupport.cs

```diff
@@ -860,6 +881,21 @@ public string GetGemma2Model()
                 helpers.DoVaeLoader( UserInput.SourceSession?.User?.Settings?.VAEs?.DefaultSD3VAE, "stable-diffusion-v3", "sd35-vae");
             }
         }
+        else if (IsFlux2() && (LoadingClip is null || LoadingVAE is null || UserInput.Get(T2IParamTypes.T5XXLModel) is not null))
```

Comment:
these extra checks aren't needed, flux.1 had a checkpoint format is why those were thing, flux.2 doesn't